R/ss3sim_base.r

```diff
@@ -208,18 +208,39 @@ ss3sim_base <- function(iterations, scenarios, f_params,
   }
   for (i in iterations) {
     # todo: fix spacing, organize comments
+    #setup ability to not run em
+    if(is.na(em_dir)){
+      em_dir <- file.path(sc, i, "em")
+      em_run <- FALSE
+    }
+    
 
     # Create folders, copy models, check for necessary files, rename
     # files for consistency
     iteration_existed <- copy_ss3models(
       model_dir = om_dir, scenarios = sc,
       iterations = i, type = "om"
     )
+  
     if (iteration_existed) next
+    if(em_run){
```

Comment:
I am thinking about implementing this in the main branch but just using the logic from line 225 instead of this if/else statement. Where, `next` would be called if `is.na(em_dir)`. Do you think that would be okay?